### PR TITLE
Add ldas-tools-al-swig

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds the `ldas-tools-al-swig` recipe, which builds SWIG bindings for the [`ldas-tools-al`](https://anaconda.org/conda-forge/ldas-tools-al/) package. Currently only the Python bindings are built, but the recipe is constructed using `outputs` to make adding other language bindings easy in the future.

cc: @emaros

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this is now ready for review. @emaros has already privately agreed to be named as a maintainer.

File: recipes/ldas-tools-al-swig/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      doc_url: https://ldastools.docs.ligo.org/LDAS_Tools/Beta/doc/ldas-tools-al/ldastoolsal/html/
+      license: GPLv2
+      license_family: GPL
+      license_file: COPYING
+      summary: Python bindings for the LDAS Tools abstraction toolkit
+
+about:
+  home: https://ldastools.docs.ligo.org/LDAS_Tools/Beta/doc/ldas-tools-al/ldastoolsal/html/
+  doc_url: https://ldastools.docs.ligo.org/LDAS_Tools/Beta/doc/ldas-tools-al/ldastoolsal/html/
+  license: GPLv2
```
Comment (marcelotrevisani) [inline recipes/ldas-tools-al-swig/meta.yaml]: It seems that you license is ``GPLv3``

Comment (duncanmmacleod) [reply recipes/ldas-tools-al-swig/meta.yaml]: @marcelotrevisani, you are correct, thanks for the correction, I fixed this in a6a6db4.

**Comment by marcelotrevisani** [comment]
LGTM

Thanks to put this in ``conda-forge``

**Pull request merged**